<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1428</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>160</y>
      <width>581</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="sectionLayout"/>
   </widget>
   <widget class="QLabel" name="imageDisplay">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>551</width>
      <height>401</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="opticalParams" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>590</y>
      <width>491</width>
      <height>171</height>
     </rect>
    </property>
    <widget class="QLabel" name="opticalLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Optical Parameters</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="opticalBtn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLabel" name="distLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Distance (cm)</string>
     </property>
    </widget>
    <widget class="QLabel" name="diamLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Diameter (mm)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="distEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="diamEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lambdaEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lambdaLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>WaveLength (nm)</string>
     </property>
    </widget>
    <widget class="QSlider" name="distSlider">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="distRealValue">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Real Value</string>
     </property>
    </widget>
    <widget class="QSlider" name="diamSlider">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="diamRealValue">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Real Value</string>
     </property>
    </widget>
    <widget class="QSlider" name="lambdaSlider">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lambdaRealValue">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Real Value</string>
     </property>
    </widget>
    <widget class="QSlider" name="intensitySlider">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="intensityLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Intensity</string>
     </property>
    </widget>
    <widget class="QLabel" name="intensityRealValue">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Real Value</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="positionParams" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>161</width>
      <height>401</height>
     </rect>
    </property>
    <widget class="QLabel" name="positionLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="positionSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>22</width>
       <height>391</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
     <property name="invertedControls">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="positionValue">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PositionVlaue</string>
     </property>
    </widget>
    <widget class="QLabel" name="ratioLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ratio</string>
     </property>
    </widget>
    <widget class="QLabel" name="ratioValue">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RatioV</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="logCheck">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>270</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Log Scale</string>
     </property>
    </widget>
    <widget class="QSlider" name="meanSlider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>22</width>
       <height>71</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="meanLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Moyenne</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="meanValue">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Moyenne</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lense">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600; background-color:transparent;&quot;&gt;LEnsE / Laboratoire d'Enseignement Expérimental&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; background-color:transparent;&quot;&gt;Démo - Diffraction / Airy&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lense_logo">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>30</y>
      <width>281</width>
      <height>121</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="cameraParams" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>590</y>
      <width>401</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QLabel" name="cameraLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Camera Parameter</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="cameraBtn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLabel" name="pixelLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Pixel Size (um)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pixelEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="originSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="originValue">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>82</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>OriginValue</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="fileBt">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QLabel" name="openFileLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NO FILE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="simulationDisplay">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>600</y>
      <width>321</width>
      <height>151</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
